### PR TITLE
Correct VREP scenes variable

### PR DESCRIPTION
**Pull request by filRocha**
Hi.
Just changed child scripts in V-REP demo scenes to the new simROS variable syntax.
Best regards.

### CODE REVIEW

**Comment by fferri** [comment]
Sorry, due to the binary format of the V-REP scene files it is not possible to review the changes.
Can you summarize the changes applied to scene files? Is it only about replacing `simExtRosInterface_` with `simROS.`?

**Comment by filRocha** [comment]
Hello, Ferri.

Yes. Basically, that was the only modification I have done in the scenes.

Best Regards.

Filipe Augusto Santos Rocha, MSc.

Pesquisador / Researcher

Instituto Tecnológico Vale / Vale Institute of Technology

Avenida Juscelino Kubitschek, 31, Bauxita, 35400-000 Ouro Preto/MG

T. 55 (31) 3552 7506 Carrier. 901 7506 M.55 (31) 97509-7144

filipe.rocha@itv.org


Em qua, 4 de jul de 2018 às 06:53, Federico Ferri <notifications@github.com>
escreveu:
